### PR TITLE
Restructuration of the website

### PR DESCRIPTION
**Pull request by Tpt**
- Moves from a student project to an open source project
- Merges documentation and modules in a "technical overview" page

Not done by this PR: update module list, ideas of improvements

### CODE REVIEW

**Comment by ProgVal** [comment]
you forgot to mention irc

**Comment by Tpt** [comment]
Will be done in an other PR

**Comment by Ezibenroc** [comment]
Great job.

**Comment by Ezibenroc** [comment]
@ProjetPP/ensl-team Something to add?

File: about.md [hunk 1]
```diff
@@ -6,8 +6,15 @@ group: navigation
 ---
 {% include JB/setup %}
 
-We are seven students in first year of the theoretical computer science master, 
-at the [École Normale Supérieure de Lyon](http://www.ens-lyon.fr/DI/?lang=en):
+The *Projet Pensée Profonde* is an open source project which aim is to create an open source question answering framework and a demo of it, [Platypus](http://askplatyp.us). Patches and new features are very welcome.
+
+You can fork the project on [GitHub](https://github.com/ProjetPP/), follow us on [Twitter](https://twitter.com/ProjetPP) and contact us by <a href="mailto:askplatyp.us">email</a>.
```
Comment (yhamoudi) [inline about.md]: Is it normal that the mailto is `askplatyp.us`?

Comment (Tpt) [inline about.md]: Fixed

**Comment by yhamoudi** [comment]
You have merged `Documentation` with `Modules`. Isn't `Modules` a bit too big now (and it will grow when we'll add docs)? Moreover, it's not clear that a "modules" page contains the documentation documents.

**Comment by Tpt** [comment]
I don't think that module is very big currently. But, yes, it may be good to split it again when we will add more content. I've kept the filename "module.md" in order to don't break urls (but we should maybe rename it).

**Comment by Tpt** [comment]
Anyone can review/fix misspelling that may remain?

**Pull request merged**